src/notebook_TempCorr.ipynb

```diff
@@ -0,0 +1,7188 @@
66687, 1951.8592293333352, 1954.242020916669, 1956.6284853333354, 1959.0239727500023, 1961.4338333333362, 1963.8634172500024, 1966.3180746666694, 1968.8031557500026, 1971.3240106666697, 1973.8859895833364, 1976.4944426666693, 1979.154720083336, 1981.8721720000033, 1984.6521485833366, 1987.5000000000034, 1990.4210764166705, 1993.4207280000037, 1996.5043049166707, 1999.6771573333372, 2002.9446354166712, 2006.3120893333376, 2009.7848692500042, 2013.3683253333384, 2017.067807750005, 2020.8886666666717, 2024.8362522500054, 2028.915914666672, 2033.133004083339, 2037.4928706666726, 2042.0008645833395, 2046.6623360000062, 2051.48263508334, 2056.467112000007, 2061.6211169166736]}, {\"hoverinfo\": \"x+y+text\", \"line\": {\"color\": \"firebrick\"}, \"name\": \"Cubic\", \"text\": \"Cubic)\", \"type\": \"scatter\", \"uid\": \"4fbcf9f3-e122-47fc-984b-a5c94a06d6db\", \"visible\": false, \"x\": [16.0, 16.1, 16.200000000000003, 16.300000000000004, 16.400000000000006, 16.500000000000007, 16.60000000000001, 16.70000000000001, 16.80000000000001, 16.900000000000013, 17.000000000000014, 17.100000000000016, 17.200000000000017, 17.30000000000002, 17.40000000000002, 17.50000000000002, 17.600000000000023, 17.700000000000024, 17.800000000000026, 17.900000000000027, 18.00000000000003, 18.10000000000003, 18.20000000000003, 18.300000000000033, 18.400000000000034, 18.500000000000036, 18.600000000000037, 18.70000000000004, 18.80000000000004, 18.90000000000004, 19.000000000000043, 19.100000000000044, 19.200000000000045, 19.300000000000047, 19.40000000000005, 19.50000000000005, 19.60000000000005, 19.700000000000053, 19.800000000000054, 19.900000000000055, 20.000000000000057, 20.10000000000006, 20.20000000000006, 20.30000000000006, 20.400000000000063, 20.500000000000064, 20.600000000000065, 20.700000000000067, 20.800000000000068, 20.90000000000007, 21.00000000000007, 21.100000000000072, 21.200000000000074, 21.300000000000075, 21.400000000000077, 21.500000000000078, 21.60000000000008, 21.70000000000008, 21.800000000000082, 21.900000000000084, 22.000000000000085, 22.100000000000087, 22.200000000000088, 22.30000000000009, 22.40000000000009, 22.500000000000092, 22.600000000000094, 22.700000000000095, 22.800000000000097, 22.900000000000098, 23.0000000000001, 23.1000000000001, 23.200000000000102, 23.300000000000104, 23.400000000000105, 23.500000000000107, 23.600000000000108, 23.70000000000011, 23.80000000000011, 23.900000000000112, 24.000000000000114, 24.100000000000115, 24.200000000000117, 24.300000000000118, 24.40000000000012, 24.50000000000012, 24.600000000000122, 24.700000000000124, 24.800000000000125, 24.900000000000126, 25.000000000000128, 25.10000000000013, 25.20000000000013, 25.300000000000132, 25.400000000000134, 25.500000000000135, 25.600000000000136, 25.700000000000138, 25.80000000000014, 25.90000000000014], \"y\": [21.94, 21.91763895833333, 21.896095000000003, 21.87534354166667, 21.85536, 21.83611979166667, 21.817598333333333, 21.79977104166667, 21.782613333333334, 21.76610
```

Comment:
Since you wrote a function for it in src/nist.py, this should be removed from the notebook and replaced by an import.

Reply:
@mathieuboudreau I removed the repetitive code by importing the function `temperature_correction` from `nist`. In the notebook, I only called the function to obtain the data for the plots.